sidecar/src/entry.rs

```diff
@@ -163,7 +173,7 @@ pub fn daemonize(listener: IpcServer, cfg: Config) -> anyhow::Result<()> {
         .map_err(|err| io::Error::new(io::ErrorKind::Other, err))
         .context("Could not spawn the sidecar daemon")?;
 
-    Ok(())
+    log_error.context("Failed to open the log file")
```

Comment:
But if you return an error here, you'll fail to connect to the sidecar in the function that called, even though subsequent calls will succeed right?

I think that's a bad idea to return the error.
* Either, it's a hard error to not be able to log to file and we shouldn't spawn the sidecar
* We admit it as acceptable an the initial call should connect

Reply:
Yes, I think you are right. It's better to push the _logging_ of what happened down here (or maybe treat certain errors as recoverable) than right out fail the first attempt.